src/covid19sim/hydra-configs/tracing-method/naive.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Name of this file is misleading... if `naive` is tristans method, we don't trace symptoms or risks.

Reply:
To be discussed in folder structuring!